modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -240,8 +240,6 @@ definition {
 	@description = "LPS-156569 - Verify that it's possible to create an Action with Groovy Script"
 	@priority = "5"
 	test CanCreateActionWithGroovyScript {
-		property portal.acceptance = "quarantine";
```

Comment:
Hey @evanilson-santana,

Next time, separate a new Commit for the remove line for example: "LRQA-XXXXX Remove  CanCreateActionWithGroovyScript of quarantine". It will help the tracking of the PR easily. Thanks!